db/src/db.rs

```diff
@@ -26,7 +26,7 @@ use entids;
 use mentat_tx::entities as entmod;
 use mentat_tx::entities::{Entity, OpType};
 use errors::{ErrorKind, Result, ResultExt};
-use types::{Attribute, AttributeBitFlags, DB, Entid, IdentMap, Partition, PartitionMap, Schema, TypedValue, ValueType};
+use types::{AVMap, AVPair, Attribute, AttributeBitFlags, DB, Entid, IdentMap, Partition, PartitionMap, Schema, TypedValue, TxReport, ValueType};
```

Comment:
Doesn't that just replace a "too wide" problem with a "too high" problem? Or maybe I'm taking you to literally?

    use stuff::{
        Long, List, Of, Things, To, Import,
        Split, Over, Several, Lines,
    };

?